wnbd-client/client.cpp

```diff
@@ -352,4 +378,15 @@ Client::Command commands[] = {
     Client::Command(
         "reset-opt", {}, "Reset driver option.",
         execute_reset_opt, get_reset_opt_args),
+    Client::Command(
+        "install-driver", {}, "Install WNBD driver and create its adapter",
+        execute_install, get_filename_args),
+    Client::Command(
+        "clean-uninstall", {}, "Remove all disk mappings with a SOFT disconnect option"
+                               " and tries to uninstall all installed WNBD drivers.",
+        execute_clean_uninstall),
+    Client::Command(
+        "force-clean-uninstall", {}, "Remove all disk mappings with a HARD disconnect option"
```

Comment:
Why not just add a "hard-disconnect" parameter, matching the "unmap" one? The timeout might become configurable as well.